stable/redmine/values.yaml

```diff
@@ -328,3 +328,103 @@ readinessProbe:
   timeoutSeconds: 1
   successThreshold: 1
   failureThreshold: 3
+
+
+## Configure CronJob scheduler for receiveing emails and converting them to tasks
+## ref: http://www.redmine.org/projects/redmine/wiki/RedmineReceivingEmails
+##
+mailReceiver:
+  ## Whether to enable scheduled task
+  enabled: false
+
+  ## CronJob schedule
+  schedule: "*/5 * * * *"
+
+  ## CronJob suspend flag
+  suspend: true
+
+  image:
+    registry: docker.io
+    repository: demonihin/redmine-mail-receiver
```

Comment:
do we need a docker image here. I was looking at https://hub.docker.com/r/demonihin/redmine-mail-receiver and it appears there is a shell script that executes rake tasks. Could it instead be possible to mount the script as a configmap and execute it in the cronjon using the bitnami/redmine image?

Reply:
I modified my PR to use bitnami/redmine. It works in my test environment.
Thank you.